frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -155,6 +155,7 @@ export default ({
         // NOTE: messagesInitiated describes if the messages are fully re-rendered
         //       according to this, we could display loading/skeleton component
         messagesInitiated: undefined,
+        renderingChatRoomId: null,
```

Comment:
Since the `this.currentChatRoomId` is a getter which can be changed anytime.
We can not use `this.currentChatRoomId` to point the current-rendering-chatRoomId because it takes some time to render the chatroom.
Since we initiate the chatroom state when we open or switch a chatroom, we can say the current-rendering-chatroom is the chatroom whose state is initiated for the last time.